documentation/modules/exploit/unix/webapp/wp_phpmailer_host_header.md

```diff
@@ -0,0 +1,62 @@
+## Intro
+
+This vuln has some caveats: you need approximately WordPress 4.6 with
+Exim for the `sendmail(8)` command.
+
+Thanks to WP's awesome practice of backporting the heck out of all their
+patches, we need to use a Git clone and check out the vuln release.
+
+## Setup
+
+This was tested on Ubuntu. YMMV.
```

Comment:
FWIW, the note about Ubuntu was for the install process, not the vulnerability. The install process shouldn't change with version.